### PR TITLE
fix: hide apps in header-menu when not logged in

### PR DESCRIPTION
**Pull request by Birkbjo**
Hide apps-menu in header bar when not logged in.

Also renames "Your apps" to "My apps", and other related menu-items, as suggested by @germanviscuso .

### CODE REVIEW

File: client/src/components/Header/Header.js [hunk 1]
```diff
@@ -65,12 +65,12 @@ const ProfileButton = () => {
...
             <Divider dense />
-            <DropdownMenuItemLink to="/user">Your apps</DropdownMenuItemLink>
+            <DropdownMenuItemLink to="/user">My apps</DropdownMenuItemLink>
             <DropdownMenuItemLink to="/user/organisations">
-                Your organisations
+                My organisations
             </DropdownMenuItemLink>
             <DropdownMenuItemLink to="/user/apikey">
-                Your API keys
+                My API keys
```
Comment (amcgee) [inline client/src/components/Header/Header.js:73]: This is probably fine for now to be consistent with `My Apps`, but interestingly GitHub uses `Your` in their profile menu...

Screen Shot 2021-12-06 at 14 28 00

Comment (amcgee) [reply client/src/components/Header/Header.js:73]: (API Keys probably should be in a general `Settings` page, eventually, but fine for now)

**Review by amcgee — APPROVED** [review]
Minor point about `My` vs `Your` but the main feature of hiding `Your Apps` when not logged in looks good to me.

@cooper-joe @varl @germanviscuso maybe we defer the renaming to `My` to a later change?  I'd also like to explore* using verbs in the main menu, i.e. `Explore` and `Manage`

File: client/src/components/Header/Header.js (hunk 1 above)
Comment (Birkbjo) [reply client/src/components/Header/Header.js:73]: Sure, we can defer the renaming. It's not really important, and will probably make our other docs/guides inconsistent.

**Pull request merged**